lib/inspec/profile.rb

```diff
@@ -465,28 +466,39 @@ def archive(opts)
       end
 
       # remove existing archive
-      File.delete(dst) if dst.exist?
+      FileUtils.rm_f(dst) if dst.exist?
       @logger.info "Generate archive #{dst}."
 
       # filter files that should not be part of the profile
       # TODO ignore all .files, but add the files to debug output
 
+      # Generate temporary inspec.json for archive
+      Inspec::JsonProfileSummary.produce_json(
+        info: info,
+        dst: "#{root_path}inspec.json",
```

Comment:
I'm suspicious there's a `/` missing from this, but I don't see where `root_path` is defined to check if it has a trailing `/` or not

Reply:
This caught me out on the first implementation, which was `"#{root_path}/inspec.json"`, when I got the error of no path `something/something//inspec.json`

Definition:   https://github.com/inspec/inspec/blob/d478064973639d7fda36e0fb10ef8d3eb9f97333/lib/inspec/profile.rb#L518

I'll double check with pry though to be sure